pkg/names/names.go

```diff
@@ -1,7 +1,36 @@
 package names
 
+import (
+	"os"
+)
+
 // ManifestDir is the directory where manifests are located.
 var ManifestDir = "./bindata"
 
 // NMStateResourceName is the name of the CR that the operator will reconcile
 const NMStateResourceName = "nmstate"
+
+// Relationship labels
+const COMPONENT_LABEL_KEY = "app.kubernetes.io/component"
+const PART_OF_LABEL_KEY = "app.kubernetes.io/part-of"
+const VERSION_LABEL_KEY = "app.kubernetes.io/version"
+const MANAGED_BY_LABEL_KEY = "app.kubernetes.io/managed-by"
+
+var RelationshipLabels map[string]string
+
+func InitRelationshipLabels() {
+	RelationshipLabels = map[string]string{}
+	mapLabelKeys := map[string]string{
+		"COMPONENT":  COMPONENT_LABEL_KEY,
+		"PART_OF":    PART_OF_LABEL_KEY,
+		"VERSION":    VERSION_LABEL_KEY,
+		"MANAGED_BY": MANAGED_BY_LABEL_KEY,
+	}
+
+	for key, label := range mapLabelKeys {
+		envVar := os.Getenv(key)
+		if envVar != "" {
+			RelationshipLabels[label] = envVar
+		}
+	}
+}
```

Comment:
I think I was wrong, since we are just reading env vars we have no need to do this at startup time, let's just read the env vars in this function and update a map instead of storing at a global var with is a worse solution, sorry about the back and forth.

Also since we have use cases where a label is already defined (enactments) is better to update a map instead of creating a new one `func IncludeRelationshipLabels(map[string]string) map[string]string`, so we can use it declaratively with a little functional programming.  

```golang
Labels: IncludeRelationshipLabels(map[string]string{shared.EnactmentPolicyLabel: policy.Name}), 
```